### DISCUSSION TITLE
这个要怎么过啊

### DISCUSSION THREAD

**Discussion by dievefess** [share121/inter-knot#2220] [其他]
![Screenshot_20240815_170100_edit_18596604525338](https://github.com/user-attachments/assets/5150ea95-0818-4b8c-88aa-8aa35999949c)
真的好难啊

**Comment by poolofwater** [comment]
把该升的升一升然后打，实在不行多打几遍？qwq

**Comment by Genshin-Neuvillette** [comment]
太难了，我对我的实力已经不抱期待了。正在为轮椅攒菲林<img src="https://bbs-static.miyoushe.com/static/2024/03/21/86aed25342a127321aa7a469f111d52d_2959126012583825986.png" alt="丽都漫步-铃" width="75" >

**Comment by anything12121** [comment]
多打打就好了，推荐打完两兄弟后先打胖子(拉扯把boss距离分开)打完胖子后拉距离打铁拳，boss有动作立马闪开，他有几个攻击不闪光且前摇小，就这样一点点偷失衡值就好了，反正任务不限时

**Comment by guozqi** [comment]
这玩意我梦回当时打14-21连续打了四个小时就在凹

**Comment by JobNumber-9527** [comment]
那个充电拳王吗？他那个手臂充电强化后，可以用其他属性打紊乱的，可以让他暂时失去强化能力<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
